FreeSurfer/custom/README

```diff
@@ -0,0 +1,7 @@
+These are scripts that can be used to get better performance out of
+FreeSurfer version 6 on high resolution data (with or without a T2
+scan). Make sure the directory that these files are in,
+${HCPPIPEDIR}/FreeSurfer/custom, is on your PATH before running
+FreeSurferPipeline.sh. Run recon-all.v6.hires as you normally would
+WITHOUT -hires or -cm but adding -conf2hires.
+
```

Comment:
This README might be a little confusing as currently written:
1) While the particular scripts in custom do not require a T2 scan, the FreeSurferPipeline.sh script itself does.
2) Perhaps delete the last sentence -- users aren't running recon-all.v6.hires directly themselves. Rather it is invoked through FreeSurferPipeline.sh (with the appropriate flags set therein).

Reply:
Looks like you've fixed that README file.